src/browser/setupWorker/glossary.ts

```diff
@@ -103,6 +103,7 @@ export interface SetupWorkerInternalContext {
   worker: ServiceWorker | null
   registration: ServiceWorkerRegistration | null
   requestHandlers: Array<RequestHandler>
+  requests: Map<string, Request>
```

Comment:
Hmm

I think we could, but it might be a bit complicated, and not worth the effort?  

I think we would need to `WeakMap<Request, {requestId, RequestClone>` to really make that change, which might be ok, but it's pretty straightforward to cleanup, and should generally be small enough at any point in time to not be an issue?